resources/js/pages/forums/topic/[topic].tsx

```diff
@@ -0,0 +1,31 @@
+import { SEO } from '@/common/components/SEO';
+import { useHydrateShortcodeDynamicEntities } from '@/common/hooks/useHydrateShortcodeDynamicEntities';
+import { usePageProps } from '@/common/hooks/usePageProps';
+import { AppLayout } from '@/common/layouts/AppLayout';
+import type { AppPage } from '@/common/models';
+import { ShowForumTopicMainRoot } from '@/features/forums/components/ShowForumTopicMainRoot';
+import type { TranslatedString } from '@/types/i18next';
+
+const ShowForumTopic: AppPage = () => {
+  const { dynamicEntities, forumTopic, paginatedForumTopicComments } =
+    usePageProps<App.Data.ShowForumTopicPageProps>();
+
+  useHydrateShortcodeDynamicEntities(dynamicEntities);
+
+  const currentPageFirstPost = paginatedForumTopicComments.items[0].body;
+  const description = `${currentPageFirstPost.slice(0, 160)}${currentPageFirstPost.length > 160 ? '...' : ''}`;
+
+  return (
+    <>
+      <SEO title={forumTopic.title as TranslatedString} description={description} />
+
+      <AppLayout.Main>
```

Comment:
Breadcrumb is escaping foreign characters:
![image](https://github.com/user-attachments/assets/03c32442-13c7-4e65-b36a-5533184b7df6)

Reply:
Surprisingly, it appears this is how the forum title is literally stored in the database (check the `forums` table). We should probably manually update the title to "RA em Português!".